Source/gg2/Global Game Settings.xml

```diff
@@ -37,6 +37,7 @@
     <writeToLog>true</writeToLog>
     <abortOnError>false</abortOnError>
     <treatUninitializedAsZero>false</treatUninitializedAsZero>
+    <checkScriptArgumentCount>false</checkScriptArgumentCount>
```

Comment:
Are we using any varargs functions?

Reply:
I really don't know, so I just decided not to touch it.

Reply:
[The same thing came up on GG2F](https://www.ganggarrison.com/forums/index.php?topic=38343.msg1297590#msg1297590) and I explained:

> I wrote several Scripts that made use of that behaviour in GM8.0 to have optional arguments. I hope GM8.1 provides a cleaner way to do it if it insists on breaking it?

And @cher-nov replied:

> It actually does. Now there's `argument_count` variable available which tellls how many arguments were passed to script.
> GM 8.1 determines the number of required arguments by parsing occurencies of `argument0`..`argument15` variables.
> If you want to use optional arguments, you should access them as `argument[0]`..`argument[15]` instead.

(Weird system but okay.)